mesonbuild/interpreter/kwargs.py

```diff
@@ -283,11 +284,11 @@ class ConfigureFile(TypedDict):
 
     output: str
     capture: bool
-    format: T.Literal['meson', 'cmake', 'cmake@']
-    output_format: T.Literal['c', 'nasm']
+    format: Literal['meson', 'cmake', 'cmake@']
+    output_format: Literal['c', 'nasm']
```

Comment:
Why removing `T.`? We use that syntax everywhere.

Reply:
`Literal` isn't in typing until Python 3.8, so as long as we support older versions of python Literal needs to come from typing_extensions.

Reply:
Ok